Services/Devices.cs

```diff
@@ -42,7 +52,8 @@ public class Devices : IDevices
         {
             try
             {
-                await this.registry.GetDeviceAsync("healthcheck");
+                await this.rateLimiting.LimitRegistryOperationsAsync(
+                    () => this.registry.GetDeviceAsync("healthcheck"));
```

Comment:
I'm assuming the device will always return null and will not throw, yes?

Reply:
interesting, this doesn't look right, I'll create a task to investigate

Reply:
https://github.com/Azure/device-simulation-dotnet/issues/97